views/fe-utils/siteTooltipContent.js

```diff
@@ -0,0 +1,30 @@
+import { N_A, TOTAL_LABEL } from '../../constants'
+import { formatAsPercentage } from './formatAsPercentageUtil'
+
+export const siteTooltipContent = (siteData, siteTotals) => {
+  const tooltipData = []
+  let siteTotalPercent = 0
```

Comment:
don't we have this already in the site data or tooltip payload?

Reply:
The data we have is the N/A percentage but that's the difference of the sites current targets. In the ticket I think we want the summation of the percentages. Maybe I'm missing something?
<img width="296" alt="Screen Shot 2022-10-03 at 3 17 44 PM" src="https://user-images.githubusercontent.com/19805355/193661518-60127424-4e7a-4b75-9ddd-bbf67b88423f.png">

Reply:
don't we have a site total count and a site total target? If so I believe we could calculate the percentage from those values.